oceannavigator/frontend/src/components/Class4Window.jsx

```diff
@@ -42,7 +42,7 @@ class Class4Window extends React.Component {
   }
 
   onLocalUpdate(key, value) {
-    if (this._mounted) {
+    if (this._mounted && value) {
```

Comment:
Need to check if `value` is undefined here because the /class4/models/ endpoint returns an empty list for the OLA files which JS interprets as undefined. This causes issues down the line when querying the Class4 OLA plots.

Reply:
JS == :facepalm: